index.rst

```diff
@@ -372,30 +372,41 @@ c.l.o stubb
 
   Here is where we currently are in the release process. Current step in bold.
 
-  Precursor Steps
+  Release Precursor Steps
   ---------------------------------
 
   1. Identify any pre-release blockers ("must-have features") :tools:
-  2. Wait for them to clear
+  Contributors check if there are outstanding issues that have to be included in the next M.m.p release and related them as blocker to the above issue DM-XXXXX.
```

Comment:
Either "next" or "M.m.p" (which will presumably be replaced with numbers), but not both.
"related" → "relate".
We still have to wait until all the blocking issues are resolved.

Reply:
added an extra step (wait until all blocking issues are resolved.)